### PR TITLE
SECURITY PAGE: Epic

### PR DESCRIPTION
**Pull request by barbaraperic**
This PR styles the Security page

---

[Figma](https://www.figma.com/design/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?node-id=1082-25121&t=eH0bHoOkI3RQXTmK-4) | [Notion](https://www.notion.so/filecoin/FF-Develop-Security-page-4e792773ac784231b9c1d09bee857f80?pvs=4)

screencapture-localhost-3000-security-2024-07-02-09_55_27
screencapture-localhost-3000-security-2024-07-02-09_55_54
screencapture-localhost-3000-security-2024-07-02-09_56_08
Screenshot 2024-07-02 at 10 09 38

### CODE REVIEW

File: src/content/pages/security.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+header:
+  title: "Filecoin Ecosystem Security"
+  description: "Filecoin Foundation is deeply committed to the integrity and security of the Filecoin network. Robust security practices are vital for developing, maintaining, and operating resilient systems – enabling the Filecoin community to innovate and grow."
```
Comment (mirhamasala) [inline src/content/pages/security.md:4]: @barbaraperic - Did we create this ourselves? Or was it provided?

File: src/content/pages/security.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+seo:
+  title: "Filecoin Ecosystem Security"
+  description: "Filecoin Foundation is deeply committed to the integrity and security of the Filecoin network. Robust security practices are vital for developing, maintaining, and operating resilient systems – enabling the Filecoin community to innovate and grow."
```
Comment (mirhamasala) [inline src/content/pages/security.md:7]: Once the page is up, we will ask David to provide an appropriate SEO title and description. Can be in the same batch as Orbit page.

File: src/app/security/page.tsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+      />
+
+      <PageSection
+        kicker="What We Do"
+        title="Incident Response and Monitoring"
+        description="Filecoin Foundation facilitates a collaborative approach to network security for the Filecoin network. This includes monitoring and network-wide responders, spanning multiple organizations and projects, to facilitate 24/7 monitoring of the chain across time zones ––setting the ecosystem apart in fostering a secure and resilient network. "
+      />
+      <PageSection kicker="Audits" title="Security Audits">
+        <CardGrid cols="smTwo">
+          {/* placeholder for the image */}
```
Comment (mirhamasala) [inline src/app/security/page.tsx:51]: Can we rely on the fallback?

File: src/app/security/page.tsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+          <div style={{ gridColumn: 'span 1' }}></div>
```
Comment (mirhamasala) [inline src/app/security/page.tsx:52]:
```suggestion
          <div className="col-span-1"></div>
```

File: src/app/security/page.tsx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+      />
+
+      <CTASection
+        title="Engage with Our Security Team"
+        description={[
+          'Filecoin Foundation security team is always looking to better understand the perspectives and needs of the Filecoin ecosystem. Feedback is invaluable! ',
+          <>
+            Don’t hesitate to contact{' '}
+            <TextLink href={FILECOIN_FOUNDATION_URLS.security.email.href}>
+              security@fil.org{' '}
```
Comment (mirhamasala) [inline src/app/security/page.tsx:111]:
```suggestion
              {FILECOIN_FOUNDATION_URLS.security.email.href}
```

File: src/content/pages/security.md (hunk 2 above)
Comment (barbaraperic) [reply src/content/pages/security.md:7]: So we leave as is for now?

File: src/app/security/page.tsx (hunk 3 above)
Comment (barbaraperic) [reply src/app/security/page.tsx:51]: You mean adding a fallback image as a placeholder?

Screenshot 2024-07-03 at 12 00 44

**Pull request merged**